config/config.go

```diff
@@ -300,6 +300,10 @@ func (c *Config) GetS3GatewayDomainName() string {
 	return viper.GetString("gateways.s3.domain_name")
 }
 
+func (c *Config) GetS3GatewayProxyURL() string {
+	return viper.GetString("gateways.s3.proxy_url")
```

Comment:
I would probaby assume a `proxy_url` is a proxy we use to talk to S3 :)
I suggest calling it `gatways.s3.fallback_url`?